plutus-core/plutus-core/src/PlutusCore/Evaluation/Machine/ExMemory.hs

```diff
@@ -94,13 +96,36 @@ type CostingInteger =
     SatInt
 #endif
 
-
 -- $(if finiteBitSize (0::SatInt) < 64 then [t|Integer|] else [t|SatInt|])
```

Comment:
What's this doing here?

Reply:
Ugh, leftover from when I was trying to do the per-platform thing with TH. Should have been deleted.

Reply:
Deleted.